kura/examples/bnd_examples/org.eclipse.kura.demo.bnd.heater/src/main/java/org/eclipse/kura/demo/bnd/heater/HeaterConfig.java

```diff
@@ -42,7 +42,7 @@
     String mode();
 
     @AttributeDefinition(
-            name = "program.startTime",
```

Comment:
Why did you change this (and all AD names)? The plugin will convert "Program start time" to "program.startTime".